WatchApp Extension/Controllers/OnOffSelectionController.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  OnOffSelectionController.swift
+//  WatchApp Extension
+//
+//  Created by Anna Quinlan on 8/20/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import SwiftUI
+import LoopCore
+
+
+final class OnOffSelectionController: WKHostingController<OnOffSelectionView>, IdentifiableClass {
+    
+    private var viewModel: OnOffSelectionViewModel = OnOffSelectionViewModel(title: "", message: "", onSelection: {_ in })
+    
+    override func awake(withContext context: Any?) {
+        if let model = context as? OnOffSelectionViewModel {
```

Comment:
[nit] If context is not a `OnOffSelectionViewModel` then we'd end up with a empty view? Consider failing here if that is the case.